protostar/testing/test_results.py

```diff
@@ -9,6 +9,13 @@
 from .test_output_recorder import OutputName
 
 
+@dataclass(frozen=True)
+class AcceptableResult:
```

Comment:
Nit: Maybe more clear would be `AcceptableTestCaseResult`? Just `AcceptableResult` does not seem to indicate what result we mean.

Reply:
There are also setup cases, test suites and some unexpected errors, that's why `AcceptableTestCaseResult` isn't general enough.